private-templates-service/client/src/components/Dashboard/PreviewModal/TemplateInfo/TemplateInfo.tsx

```diff
@@ -1,19 +1,18 @@
 import React from 'react';
-
-import { RootState } from '../../../../store/rootReducer';
 import { connect } from 'react-redux';
+import { RootState } from '../../../../store/rootReducer';
+import { updateCurrentTemplateVersion } from '../../../../store/currentTemplate/actions';
+import { ActionButton, IDropdownOption } from 'office-ui-fabric-react';
+import { withRouter, RouteComponentProps } from 'react-router-dom';
+import { Template, TemplateInstance, PostedTemplate } from 'adaptive-templating-service-typescript-node';
```

Comment:
can we organize the imports so that all redux imports and all UI fabric imports are grouped together?